lib/auth.coffee

```diff
@@ -29,6 +30,23 @@ getAuth = (deps, opts) ->
 	###
 	exports.twoFactor = twoFactor
 
+	userDetails = null
+	exports.getUserDetails = ->
+		return new Promise (resolve, reject) ->
+			if(userDetails)
+				resolve userDetails
```

Comment:
Nit: `if` should have a space, and `resolve` should have brackets (we only skip brackets for multi-line function calls)